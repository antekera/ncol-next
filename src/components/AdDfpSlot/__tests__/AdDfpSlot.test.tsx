import React from 'react'

import { render } from '@testing-library/react'

import { AdDfpSlot } from '..'

describe('AdDfpSlot', () => {
  test('should be defined', () => {
    const { container } = render(<AdDfpSlot />)
    expect(container.firstChild).toBeDefined()
  })
})
