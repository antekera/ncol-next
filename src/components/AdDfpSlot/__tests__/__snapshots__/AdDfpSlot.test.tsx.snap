// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdDfpSlot should match snapshots 1`] = `
<div
  class="bloque-adv"
>
  <div
    class="ad-slot"
    id="12456789-0"
    style="width: inherit; height: inherit;"
  />
</div>
`;
